presto-docs/src/main/sphinx/release/release-0.193.rst

```diff
@@ -0,0 +1,76 @@
+=============
+Release 0.193
+=============
+
+General Changes
+---------------
+
+* Add :func:`map_zip_with` lambda function.
+* Add :func:`normal_cdf` function.
+* Add ``SET_DIGEST`` type and related functions.
+* Fix ``Tried to free more revocable memory than is reserved`` error for queries
+  that enable spilling and that are moved to the reserved memory pool.
+* Fix a bug that can cause the planner to go into an infinite loop. This bug was affecting queries
+  with multiple predicates that couldn't be pushed down to the connectors.
+* Fix a bug that may cause queries with ``row_number()`` to fail or return incorrect results.
+  If the query had an equality predicate against ``-1`` or ``0`` the bug would cause the query to fail.
+  If the query had a less-than-or-equal-to predicate against a negative value the bug would cause
+  the query to produce incorrect results.
+* Fix query analysis failure when lambda is used in ``UNNEST`` or ``VALUES``.
+* Add the peak of total non-revocable memory reservation (user + system memory) to query stats.
+* Add an optimizer rule that rewrites table scans in queries against partitioned tables to an empty relation
+  when the query has predicate(s) that do not match any partitions of the table.
+* Improve the performance of the :func:`ST_Contains` function.
+* Turn on new local scheduling algorithm by default (described in 0.181 release notes).
+* Remove the ``information_schema.__internal_partitions__`` table. This table was only useful
+  for the Hive connector and it became obsolete after ``SHOW PARTITIONS`` has been improved.
+
+Security Changes
+----------------
+
+* When using multiple authentication types the authentication methods are now applied in the order
+  they are specified in the ``http-server.authentication.type`` configuration.
+
+CLI Changes
+-----------
+
+* Fix CLI to abort the query when the pager exits.
+* Fix rendering of maps of Bing tiles in the CLI.
+
+JDBC Driver Changes
+-------------------
+
+* Use SSL by default for port 443.
+
+Hive Changes
+------------
+
+* Fix a bug that causes a ``ConcurrentModificationException`` when dropping any column
+  other than the last one.
+* Fix a bug that causes a newly added partition to have incorrect on-disk data format
```

Comment:
What's "non-default text format"?

Reply:
/cc @findepi

Reply:
IIUC this bug affects only the text file format. So I think we can just say `Fix a bug that causes a newly added partition to have incorrect on-disk data format for tables using the text file format`.

The related issue and PR are:
- https://github.com/prestodb/presto/issues/7301
- https://github.com/prestodb/presto/pull/9784

Reply:
Yup. It was about text format, in case when non-default format properties (e.g. field delimiter) were supposed to be used.

Reply:
This needs to be more specific:

* Correctly write files for text format tables that use non-default delimiters. Previously, they were written with the default delimiter.